mypy/report.py

```diff
@@ -331,12 +331,10 @@ def on_file(self,
             for lineno, line_text in enumerate(input_file, 1):
                 status = visitor.line_map.get(lineno, stats.TYPE_EMPTY)
                 file_info.counts[status] += 1
-                if line_text.endswith('\n'):
-                    line_text = line_text[:-1]
                 etree.SubElement(root, 'line',
                                  number=str(lineno),
                                  precision=stats.precision_names[status],
-                                 content=line_text)
+                                 content=line_text.strip('\n'))
```

Comment:
Well, it should still be the same, since we know we are always processing one line. But `rstrip()` is probably better stylistically.

Originally, I was worried about removing legitimate newlines in the code. But they will be escaped and therefore won't be removed.